.github/workflows/docker.yml

```diff
@@ -1,13 +1,9 @@
 name: Build Docker
-# Rebuilds the tendermint docker image on every push to default branch and creation of tags.
+# Rebuilds the tendermint docker image on every push to default branch.
 on:
-  pull_request:
   push:
     branches:
       - v[0-9]+.[0-9]+.x-celestia
-    tags:
-      - "v[0-9]+.[0-9]+.[0-9]+" # Push events to matching v*, i.e. v1.0, v20.15.10
-      - "v[0-9]+.[0-9]+.[0-9]+-rc*" # Push events to matching v*, i.e. v1.0-rc1, v20.15.10-rc5
```

Comment:
The issue description did say "only for pushes to default branch," but that was in the context of "pushes to pull requests and pushes to the default branch." We actually want this job to continue running on tags as well.

Reply:
Thanks for clarifying @adlerjohn, updated the PR.